docs/wiki/development/coding-guidelines.md

```diff
@@ -178,6 +178,10 @@ Every function should have a header of the following format as the start of thei
  * Arguments:
  * 0: The first argument <STRING>
  * 1: The second argument <OBJECT>
+ * 2: Multiple input types <STRING|ARRAY|CODE>
+ * 3: Optional input <BOOL> (default: true)
+ * 4: Optional input with multiple types <CODE|STRING> (default: {true})
+ * 5: Not mandatory input <STRING> (default: nil)
```

Comment:
This is the only syntax I'm not sure about here, I think default should always be an actual value.

Reply:
What about e.g.: [fnc_getWeaponState](https://github.com/acemod/ACE3/blob/master/addons/common/functions/fnc_getWeaponState.sqf)

Reply:
Yeah I think `(optional)` is bad, I guess maybe it would be better as `None` instead of `nil` to be consistent with functions which return `nil` or have no input parameters

Reply:
🤔 Actually, `None` doesn't really fit right in this context. I think you're right to go with `nil` on consideration.

Reply:
I must add, this is a very special case. I think I've found only 2 functions, where this could be used inside ace.
It's more like to have this case defined instead of using it on a regular basis.